articles/dev-itpro/deployment/infrastructure-stack.md

```diff
@@ -44,6 +47,9 @@ Customers using the self-service capabilities will see the following changes in
 
 - You will no longer have remote desktop access to the Tier 2+ sandbox environments. All operations that need remote desktop access are now available as self-service actions. The following image shows some of the operations in the environment’s **Maintain** \> **Move database menu option**. 
 
+[!Important]
+Remote desktop access will not be avaiable only to environments deployed using the self-service deployment. There is no change to the existing environments.
```

Comment:
Remote desktop access will be restricted only for environments deployed using self-service deployment. There is not change to existing environments or existing customers.